### PR TITLE
Bfu teamdash

### PR DESCRIPTION
**Pull request by B1s9l**
# Pull Requests Checklist
 
Assignee: B1s9l
 
- \[x\] code commented?
- \[x\] documentation added / updated?
- \[x\] pre-commit checks passed?
- \[x\] pipelines passed?
 
Reviewer: Monti934
 
- \[x\] are there any code smells?
- \[x\] did pipeline pass?
- \[x\] is the code well documented?

### CODE REVIEW

**Review by Monti934 — APPROVED** [review]

File: src/components/routing/routeProtectors/LoginGuard.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const LoginGuard = () => {
     return <Outlet />;
```
Comment (5v3nn) [inline src/components/routing/routeProtectors/LoginGuard.js:12]: @B1s9l make api login request. i could just set an arbitrary token value and then i am logged in, which is bad.

maybe we need to think about an `/auth` endpoint to check authorization of local auth token

File: src/components/views/Registration.tsx [hunk 2]
```diff
@@ -42,16 +53,16 @@ const Registration = () => {
   const doRegister = async () => {
     try {
       const requestBody = JSON.stringify({ username, name, password });
-      const response = await api.post("/users", requestBody);
+      const response = await api.post("/register", requestBody);
```
Comment (5v3nn) [inline src/components/views/Registration.tsx:56]: @B1s9l  error handling. depending on return status, give different hint to user. not just "Something went wrong"

File: src/components/views/TeamsOverview.tsx [hunk 3]
```diff
@@ -6,15 +6,19 @@ import { Button } from "components/ui/Button";
...
+  const [isCreateTeamOpen, setCreateTeamOpen] = useState(false);
 
   useEffect(() => {
     const fetchUserTeams = async () => {
       try {
-        const response = await api.get("/user/teams");
+        const response = await api.get("/teams");
         setUserTeams(response.data);
       } catch (error) {
         console.error("Error fetching user teams:", error);
```
Comment (5v3nn) [inline src/components/views/TeamsOverview.tsx:24]: @B1s9l improve error handling

File: src/components/views/TeamsOverview.tsx [hunk 4]
```diff
@@ -24,38 +28,56 @@ const TeamsOverview = () => {
     fetchUserTeams();
 
     const fakeUserTeams = [
```
Comment (5v3nn) [inline src/components/views/TeamsOverview.tsx:30]: @B1s9l mark what is still WIP

**Review by 5v3nn — COMMENTED** [review]
@B1s9l @Monti934 Not done

**Pull request merged**